src/lib/__tests__/api-test.js

```diff
@@ -27,7 +36,11 @@ describe('API helper', () => {
                 data,
             };
 
-            expect(api.parseResponse(response)).to.equal(data);
+            sandbox.stub(api, 'client').resolves(response);
+
+            return api.get(url).then(returnedData => {
```

Comment:
Minor, but we could standardize these to `.then(result =>` or something similar.